pkg/pillar/base/logobjecttypes.go

```diff
@@ -0,0 +1,443 @@
+// Copyright (c) 2020 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package base
+
+import (
+	"encoding/json"
+	"github.com/satori/go.uuid"
+	log "github.com/sirupsen/logrus"
+)
+
+// ObjectType : Predefined object types
+type ObjectType string
```

Comment:
I think "Object" is getting overloaded here, since you use that as part of LogObjectType. Makes the reader ask what is the difference.

How about naming this thing a "LogEventType" with the three values being LogUnknownEventType, LogObjectEventType="object",  and LogRelationEventType="relation"?

We can even drop the "Event" part of those names. Thoughts?

Reply:
dropping Event creates clash between LogObjectType type and const variable names. Will use LogObjectEventType & LogRelationEventType.